autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -660,6 +677,16 @@ def on_graph_execution(
             log_metadata.info(
                 f"⚙️ Graph execution #{graph_exec.graph_exec_id} is already running, continuing where it left off."
             )
+        elif exec_meta.status == ExecutionStatus.WAITING_FOR_REVIEW:
+            exec_meta.status = ExecutionStatus.RUNNING
+            log_metadata.info(
+                f"⚙️ Graph execution #{graph_exec.graph_exec_id} was waiting for review, resuming execution."
+            )
+            update_graph_execution_state(
+                db_client=db_client,
+                graph_exec_id=graph_exec.graph_exec_id,
+                status=ExecutionStatus.RUNNING,
+            )
```

Comment:
So we find all queued status nodes to be added here to be added into the queue correct. now we have added this case, should we also include the one waiting_for_review then ? because if it's already have the data we can proceed.